tests/test_uflash.py

```diff
@@ -382,7 +393,7 @@ def test_extract_paths():
 
     with mock.patch('uflash.extract_script', mock_e) as mock_extract_script, \
             mock.patch('builtins.print') as mock_print, \
-            mock.patch('builtins.open', mock_o) as mock_open:
+            mock.patch.object(builtins, 'open', mock_o) as mock_open:
```

Comment:
To make it work in Python 2 :D 
(but I /think/ it needs to use patch.object because it returns an object that we need to mock too).
In fact that possibly means I can simplify the setup of that mock_o object. I'll investigate.